frontend/src/main/scala/bloop/util/JavaCompat.scala

```diff
@@ -20,4 +20,11 @@ object JavaCompat {
   implicit class EnrichOption[T](option: Option[T]) {
     def toOptional: Optional[T] = InterfaceUtil.toOptional(option)
   }
+
+  implicit class EnrichSAM[T, R](f: T => R) {
+    import java.util.function.Function
+    def toJava: Function[T, R] = new Function[T, R] {
+      def apply(t: T): R = f(t)
+    }
+  }
```

Comment:
Wow why is that required?

Reply:
That was required because there are some limitations in which you cannot convert directly a function type to a SAM, for example when you get a function as a parameter and you place it somewhere where the JVM expects a SAM (`computeIfAbsent`). However, there was only one use site of this, so I've done an in-place function type, and *that one* will indeed be samified. So I'm removing this piece of code.